tests/acceptance/expectUserAndAdminLoginCest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ Cest is a class based format. Cest format is able to support multiple 
+ tests per file and easy reuse of code by adding additional private functions.
+ Cest combines scenario-driven test approach with OOP design. 
+ */
+class expectUserAndAdminLoginCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        //runs before each test
+        $I->amOnPage('/');
+    }
+    public function _after(AcceptanceTester $I)
+    {
+        //runs after each test
+    }
+    // tests
+    
+    public function expectsUserToLogin(AcceptanceTester $I) {
+        $I->wantTo('log in as a user');
+        $I->loginAs('chappress', 'password');
```

Comment:
I just tested it twice. Once by changing it to `$I->loginAs('chappresss', 'fail');` and `$I->loginAs('random', 'password');`

It passed both times. I'll look into why it doesn't fail when credentials are wrong and fix it so that it only passes with the correct credentials.